app/classes/document_query.rb

```diff
@@ -31,6 +31,7 @@ def body
       search.suggest(:suggestion, suggestion_hash)
     end
     build_search_query
+    search.explain true if Rails.logger.level == 0 #scoring details
```

Comment:
I had to look up that level 0 is `:debug`. How would you feel about using `Rails.logger.debug?` instead of `Rails.logger.level == 0`?

Also, I just checked production, and we use logger level `:debug` there. Whoops! Would you mind changing it to `:info` or `:warn` for production as part of this commit? I'm not sure if there's any historical reason for why we're using the `:debug` level in production.

```
[search@services1.us-east-1 current]$ RAILS_ENV=production bundle exec rails c
Loading production environment (Rails 5.1.2)
irb(main):001:0> Rails.logger.level
=> 0
```

Reply:
Ah - I tried `Rails.logger.level.debug?` originally, which didn't work - duh. Will change. I will also adjust the logger level to `:info` for production.